_includes/about.html

```diff
@@ -10,5 +10,10 @@
       across the world who can watch, participate and share their views
       right from the comfort of their homes.
     </div>
+    <div class="content">
+      <a href="/2020/about" class="primary-button">
```

Comment:
Can you please make it a relative URL?

Ref: https://talk.jekyllrb.com/t/relative-url-and-baseurl/2051/6

Reply:
Sure

Reply:
Done